src/cli/platform/sgx/crl.rs

```diff
@@ -22,10 +31,47 @@ pub struct CrlCache {}
 
 impl CrlCache {
     pub fn execute(self) -> anyhow::Result<ExitCode> {
-        let dir = sgx_cache_dir()?;
-        save_file(CERT_CRL, &dir)?;
-        save_file(PROCESSOR_CRL, &dir)?;
-        save_file(PLATFORM_CRL, &dir)?;
+        let mut dest_file = sgx_cache_dir()?;
+        dest_file.push("crls.der");
+
+        let dest_file = Path::new(&dest_file);
+        if dest_file.exists() {
+            std::fs::remove_file(dest_file)
```

Comment:
No need to remove the file, if you use `truncate` in `OpenOptions`

Reply:
Done